evdev/input.c

```diff
@@ -60,8 +65,8 @@ device_read(PyObject *self, PyObject *args)
         return NULL;
     }
 
-    PyObject* sec  = PyLong_FromLong(event.time.tv_sec);
-    PyObject* usec = PyLong_FromLong(event.time.tv_usec);
+    PyObject* sec  = PyLong_FromLong(event.input_event_sec);
```

Comment:
This looks suspicious to me. The whole point of `input_event_sec` is that it is an **unsigned** long. And now you're casting it back to a long. Are you sure this is correct?